### ISSUE TITLE
problems building vbz_compression target

### ISSUE THREAD

**Issue by nadyawilliams** [giesselmann/nanopype#21]
### Prerequisites

* [x ] Put an X between the brackets on this line if you have done all of the following:
    * Read the contributions guide at: <https://nanopype.readthedocs.io/en/latest/contribute>
    * Checked that your issue isn't already filed: <https://github.com/giesselmann/nanopype/issues>

### Steps to Reproduce

1. per instructions on  https://nanopype.readthedocs.io/en/latest/installation/src/
    create nanotype conda environment and checked out Nanopype 
    conda create -n nanopype python=3.6 anaconda
    source activate nanopype
    git clone https://github.com/giesselmann/nanopype
    cd nanopype
    git fetch --tags && git checkout v1.0.0
    python3 -m pip install --upgrade pip
    python3 -m pip install -r requirements.txt
    cd ..
2. Running snamemake commands there is en error when building vbz_compression.
    Command:  snakemake --snakefile rules/install.smk --directory ~/nanopype  vbz_compression
    Partial Error:

```
    [Thu Apr 15 18:24:28 2021]
    rule vbz_compression:
        output: lib/libvbz_hdf_plugin.so.1.0.1
        jobid: 0
    fatal: destination path 'vbz_compression' already exists and is not an empty directory.
    CMake Error: The source directory "/data/homezvol1/courtnkc/nanopype/src" does not appear to contain CMakeLists.txt.
    Specify --help for usage, or press the help button on the CMake GUI.
[Thu Apr 15 18:24:28 2021]
Error in rule vbz_compression:
    jobid: 0
    output: lib/libvbz_hdf_plugin.so.1.0.1
    shell:
        install_prefix=`pwd`
        mkdir -p lib
        mkdir -p src && cd src
        if [ ! -d singularity ]; then
            git clone https://github.com/nanoporetech/vbz_compression --recursive --branch v1.0.1 --depth=1 && cd vbz_compression
        else
            cd vbz_compression && git fetch --all --tags --prune && git checkout tags/v1.0.1
        fi
        rm -rf build
        mkdir build && cd build
        cmake -D CMAKE_BUILD_TYPE=Release -D ENABLE_CONAN=OFF -D ENABLE_PERF_TESTING=OFF -D ENABLE_PYTHON=OFF ..
        make
        cp bin/libvbz_hdf_plugin.so ${install_prefix}/lib/libvbz_hdf_plugin.so.1.0.1
        
        (exited with non-zero exit code)
```

   This error results from a typo in **nanopype/rules/install.smk** file.  For the  "rule vbz_compression:" target
    lIne  

```if [ ! -d singularity ]; then```
should be changed to 
```if` [ ! -d vbz_compression ]; then```

   Then the closing of the repo works properly.

3. Once above typo is corrected, a new error indicates that the build process for vbz_compression
   cant find libzstd library during  running cmake command. Partial error :

```
        -- HDF5: Using hdf5 compiler wrapper to determine C configuration
        -- Found HDF5: /data/homezvol1/.conda/envs/nanopype/lib/libhdf5.so;/usr/lib64/librt.so;/usr/lib64/libpthread.so;/
data/homezvol1/.conda/envs/nanopype/lib/libz.so;/usr/lib64/libdl.so;/usr/lib64/libm.so (found suitable version "1.10.4", 
minimum required is "1.8.16")  
        CMake Error at /opt/foundation/share/cmake-3.12/Modules/FindPackageHandleStandardArgs.cmake:137 (message):
        Could NOT find zstd (missing: ZSTD_LIBRARY ZSTD_INCLUDE_DIR) (Required is
        at least version "1.3.1")
        Call Stack (most recent call first):
        /opt/foundation/share/cmake-3.12/Modules/FindPackageHandleStandardArgs.cmake:378 (_FPHSA_FAILURE_MESSAGE)
        cmake/Findzstd.cmake:36 (find_package_handle_standard_args)
        CMakeLists.txt:139 (find_package)
```
  The need library does exist in the user's conda nanopype environment and is at acceptable higher version 1.4.5
     Please note, that the came conda nanopype environment provides HDF5 library and it is found just fine, yet the 
     libzstd.so  in the same directory is not found. 
    
 To mitigate,  I had to manually set

```
     export CMAKE_INCLUDE_PATH=$CONDA_PREFIX/include:$CMAKE_INCLUDE_PATH
     export CMAKE_LIBRARY_PATH=$CONDA_PREFIX/lib:$$CMAKE_LIBRARY_PATH
```
 
 After that the compilation proceeded without an error. 

**Expected behavior:**

Some prerequisites are provided via system RPMS, some from the conda environment. None were missing.
The install.smk file should take into account conda environment and set appropriate definitions 
for cmake or any other commands. One  should not need to patch environment when running tools installation. 

Please also consider that many users who need to install nanotype are not software developers or installers
and  cant really debug many of the steps  of this install. A proper check for the prerequisites fulfilled and
better error handling d would be helpful to such users. 

### Versions

whatever is in the master branch currently for nanopype 1.1.0

### Log Files

* [ ] Put an X between the brackets on this line if you have attached log files from the observed behavior.

**Comment by giesselmann** [comment]
Hi,
Thank you for reporting the bug and providing a fix! I'll include it in the next version.
The path adjustments can be handled in the install.smk Snakefile to make conda builds more robust.
Thank you,
Pay

**Comment by giesselmann** [comment]
Should be fixed in v1.2.0
Please re-open if the issue remains.
